test/core/tsi/alts/handshaker/alts_concurrent_connectivity_test.cc

```diff
@@ -105,11 +105,15 @@ class FakeHandshakeServer {
     int port = grpc_pick_unused_port_or_die();
     address_ = grpc_core::JoinHostPort("localhost", port);
     if (check_num_concurrent_rpcs) {
-      service_ = grpc::gcp::
-          CreateFakeHandshakerService(kFakeHandshakeServerMaxConcurrentStreams /* expected max concurrent rpcs */);
+      service_ = grpc::gcp::CreateFakeHandshakerService(
+          kFakeHandshakeServerMaxConcurrentStreams /* expected max concurrent
+                                                      rpcs */
+          ,
```

Comment:
The comma should move to before the `/*`.

Reply:
done, removed comment since the `kFakeHandshakeServerMaxConcurrentStreams` constant seemed descriptive.